app/modules/ws2812.c

```diff
@@ -51,7 +51,7 @@ static void ICACHE_RAM_ATTR ws2812_write(uint8_t *pixels, uint32_t length) {
   // 0b00000111 => 000111 => [0]111000[1] => 10001110 => 01
   // 0b00110100 => 110100 => [0]001011[1] => 11101000 => 10
   // 0b00000100 => 000100 => [0]001000[1] => 11101110 => 11
-  uint8_t _uartData[4] = { 0b00110111, 0b00000111, 0b00110100, 0b00000100 };
+  static const uint8_t _uartData[4] = { 0b00110111, 0b00000111, 0b00110100, 0b00000100 };
```

Comment:
I took the time to dig into this a bit further and found that the array contents will be placed into .rodata section in both cases. Thus no repeated initialization of the local variable on the stack - the complier is smart enough to determine that `_uartData` is never written to. So much for the good news.

The bad news is summarized in #504:
> The tradeoff is that 8bit and 16bit const data access gets a fair bit slower.

At the moment I see no other way to avoid this by mapping the array explicitly to the .data section like this:
```c
static const __attribute__((section(".data._uartData"))) uint8_t _uartData[4] = { 0b00110111, 0b00000111, 0b00110100, 0b00000100 };
```
Maybe there's a smarter approach.